src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -153,6 +154,7 @@
             "The provided ${fieldName} is not acceptable to TEAMMATES as it cannot be empty.";
 
     // field-specific error messages
+    public static final String GRACE_PERIOD_NEGATIVE = "Grace period should never be negative.";
```

Comment:
never -> not

Reply:
Also, move this to after the time zone related messages. Can also add a hint: `The value must be one of the options in the grace period dropdown selector.` (actually, you might want to validate that too, though not as important)